GW2EIEvtcParser/EIData/ProfHelpers/ProfHelper.cs

```diff
@@ -25,6 +25,46 @@ internal static class ProfHelper
             new EffectCastFinderByDst(RuneOfNightmare, EffectGUIDs.RuneOfNightmare).UsingOrigin(InstantCastFinder.InstantCastOrigin.Gear),
             new BuffGainCastFinder(PortalEntranceWhiteMantleWatchwork, PortalWeavingWhiteMantleWatchwork),
             new BuffGainCastFinder(PortalExitWhiteMantleWatchwork, PortalUsesWhiteMantleWatchwork).UsingBeforeWeaponSwap(true),
+            // Relics
+            new BuffGainCastFinder(RelicOfVass, RelicOfVass).UsingOrigin(InstantCastFinder.InstantCastOrigin.Gear),
+            new BuffGainCastFinder(RelicOfTheFirebrand, RelicOfTheFirebrand).UsingOrigin(InstantCastFinder.InstantCastOrigin.Gear),
+            new BuffGainCastFinder(RelicOfIsgarrenTargetBuff, RelicTargetPlayerBuff).UsingChecker((bae, combatData, agentData, skillData) =>
```

Comment:
The RelicTargetPlayerBuff is unreliable, it does not refresh as the RelicOfXXXTargetBuff gets refreshed on the target.

Reply:
i see both being repplied each time, am i doing something wrong?
![5840](https://github.com/baaron4/GW2-Elite-Insights-Parser/assets/23245218/a2e3acd7-f5b8-4716-bc0a-51ac057fb710)
[peitha.zip](https://github.com/baaron4/GW2-Elite-Insights-Parser/files/12448279/peitha.zip)

Reply:
It does not get reapplied to you if the target has still the buff running, only on new applications, at least that was the case for dragonhunter relic when we tested it with Zerthox.

Reply:
https://discord.com/channels/456611641526845473/1139667440465354752/1143698177447448576

Reply:
peitha has a projectile effect so i switched to using that, i'll try to produce what was said for isgarren and dragonhunter